subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -559,16 +570,10 @@ For instance, a `ServiceRegistryProcessor` could create a single `ServiceRegistr
 
 Aggregating processors have the following limitations:
 
-
-* Its annotations need to have `CLASS` or `RUNTIME` retention
-* Can't read resources (this may change in the future)
-* Can't write resources (this may change in the future)
-* Can't have any side effects except for using the `Filer` and `Messager` APIs
-* Can't depend on compiler-specific APIs like `com.sun.source.util.Trees`
-* Can't depend on information only available from source files (See comment on parameter names below)
+* Can only read `CLASS` or `RUNTIME` retention annotations
+* Can only read parameter names if the user passes the `-parameters` compiler argument.
```

Comment:
Again, "They can only read...". The alternative is to change the lead-in sentence:

    Aggregating processors can only:

     * read `CLASS` or `RUNTIME` retention annotations
     * read parameter names if the user passes the `-parameters` argument to the compiler